mirrord/layer/tests/common/mod.rs

```diff
@@ -741,6 +741,7 @@ pub enum Application {
     RustRecvFrom,
     RustListenPorts,
     Fork,
+    /// `mirrord/layer/tests/apps/readlink/readlink.c`
```

Comment:
:eyes:

Reply:
Just thought it would be nice to have the test files there. Should I remove or may I keep it?

Reply:
Nah, at that point I've already done my `gr` command to see which file it is.